tests/integration.rs

```diff
@@ -6112,7 +6112,7 @@ mod tests {
             workload_path.push("workloads");
 
             let mut ovmf_path = workload_path.clone();
-            ovmf_path.push("OVMF.fd");
+            ovmf_path.push("OVMF-4b47d0c6c8.fd");
```

Comment:
There is already a `WINDOWS_IMAGE_NAME` in this file. I think it is better to define a `OVMF_IMAGE_NAME` or something to avoid modifying multiple places.

Reply:
The CI has passed this can of course be done in a follow-up PR. There is no need to block progress for such a trivial change.

Reply:
(FYI @liuw I incorporated your feedback)

Reply:
Thank you.